src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState, useRef } from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+import {
+  closeSideNav as closeSideNavAction,
+  openSideNav as openSideNavAction,
+  pinMenuTrigger as pinMenuTriggerAction,
+  unpinMenuTrigger as unpinMenuTriggerAction,
+} from '../actions';
+
+const MobileMenuTrigger = ({
+  closeSideNav,
+  focusTriggerButton,
+  openSideNav,
+  isMenuTriggerPinned,
+  pinMenuTrigger,
+  unpinMenuTrigger,
+}) => {
+  const button = useRef(null);
+  const placeholder = useRef(null);
+
+  const [mobileBreakpoint] = useState(767);
```

Comment:
Can we have `767` as a constant somewhere?

Reply:
Yeah we totally should. Not sure where to put it though. This is effectively the `$medium-screen` breakpoint (actually that one is 768) defined in the design system (https://design.va.gov/design/breakpoints) and I'd love to pull it in from there somehow. But for now I'll just move it to a const in this file if that sounds good to you.

Reply:
I moved it to the constants file and it's an improvement. Thanks!